test/test.js

```diff
@@ -20,6 +22,12 @@ const SEPARATOR = '\u001F';
 test.beforeEach(async () => pify(exec)(`rm -rf ${dest} ${destUnpacked}`));
 
 test('equals to sample', async t => {
+  // 1482680798652 - datetime for test/fixtures/output.apkg
```

Comment:
I'll remove. But after I'll got situation where I have ```const now = 1482680798652;``` with some magic-number without any comment about it's nature. 

Not cool for me